test/models/host/subscription_facet_test.rb

```diff
@@ -215,6 +215,71 @@ def test_update_foreman_facts
       assert_includes values.map(&:name), '_timestamp'
     end
 
+    def test_update_foreman_facts_with_centos_no_minor_version
```

Comment:
nitpick: our unwritten convention is test_<method_name>_<use_case> so I'd prefer `test_update_facts_with_centos_no_minor_version`